templates/cf-apache.service.mustache

```diff
@@ -2,6 +2,7 @@
 Description=CFEngine Enterprise Webserver
 After=syslog.target
 After=cf-postgres.service
+Requires=cf-postgres.service
```

Comment:
I don't see this in `core/misc/systemd/cf-apache.service`.

```
> $ diff -u templates/cf-apache.service.mustache ../core/misc/systemd/cf-apache.service | grep Requires
-Requires=cf-postgres.service
                                       
```

Reply:
It is there in core since Feb 19: https://github.com/cfengine/core/commit/ce2862bfc3f8c864c5335d25dd9a6ff15cfe120a

Reply:
The `Requires=cf-postgres.service` in `cf-apache.service` is still there in master as of today (2021-06-29).

Reply:
I don't think this should be `Requires` but rather `Wants` as it is in `cf-hub.service`. I will change and test.